src/methodology-modal/MethodologyModal.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { HeadingDescription, HeadingTitle } from "../heading";
+import Modal from "../modal";
+
+const MethodologyWrapper = styled.div``;
+const MethodologyHeader = styled.div`
+  margin-bottom: 48px;
+`;
+const MethodologyHeading = styled(HeadingTitle)``;
+
+const MethodologyDescription = styled(HeadingDescription)`
+  font-size: 16px;
+`;
+
+const MethodologyBody = styled.div``;
+
+const SectionTitle = styled.h1`
```

Comment:
Note to self: This can also inherit from `HeadingTitle`, but provide it overrides as necessary